product/host.go

```diff
@@ -16,6 +16,11 @@ import (
 	"github.com/hashicorp/hcdiag/runner"
 )
 
+const (
+	TimeoutTenSeconds    = runner.Timeout(10 * time.Second)
+	TimeoutThirtySeconds = runner.Timeout(30 * time.Second)
```

Comment:
I wonder if we should have a single default timeout for host runners or if we actually need multiple. And, perhaps we could give it a semantic name, so it was clearer why it is, rather than what it is. Perhaps something like:
```suggestion
	DefaultHostTimeout    = runner.Timeout(30 * time.Second)
```

Reply:
I want to come back to potential shared default timeouts once we've added a bunch of runner-specific defaults and know more about each runner. Using a longer shared time is more what i would expect from a deadline set at the agent or product context. Having tighter runner-specific timeouts makes retries with timeouts possible.